SConstruct

```diff
@@ -39,10 +39,13 @@ env = Environment(CXXFLAGS="-std=c++11 " + FFLAGS + OPTFLAGS + WFLAGS)
 
 comm_env = env.Clone()
 comm_env.Replace(
-        CPPPATH = ['include', 'src'],
+        CPPPATH = ['include', 'src',
+                   os.getenv('NLOHMANN_JSON_INCLUDE', default='/usr/include')],
         LIBS    = [],
         LIBPATH = []
              )
+comm_env.Append
```

Comment:
not sure why this is here

Reply:
Possibly because they wanted to keep same tamplate for all SCons* files, but for this package, both the library names (`LIBS`) and paths to them (`LIBPATH`) are empty, i.e. no other libs are needed. We can remove both of them here, but if in the future we need to add a `'.so` or `.a`, then we would be wondering for a moment where to specify those.

Reply:
ok cool, but it is only needed if you actually need to append stuff. This is well documented, we don't need to leave there.

Reply:
I just noticed it is a 'naked' Append, without any parentheses. Must be a left-over. Will remove or fix it.